config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Chess::Application.routes.draw do
   devise_for :users
   root 'home_pages#index'
+  resources :games, only: :create
 
   resources :games do
```

Comment:
I think line 4 is redundant. I added the full "resources" on line 6/7 since I think we will need to be able to update (and maybe delete?) games (as well as pieces). If they're too much and we want create only, though, we should take 6 & 7 out.